src/site/_transforms/responsive-images/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview An eleventy transform that rewrites image paths to use our
+ * image CDN.
+ */
+
+const cheerio = require("cheerio");
+const {determineImagePath} = require("./helpers");
+
+const responsiveImages = (content, outputPath) => {
+  const $ = cheerio.load(content);
+  const $img = $("img");
+  $img.each((i, elem) => {
+    const $elem = $(elem);
+    const originalSrc = $elem.attr("src");
+    const newSrc = determineImagePath($elem.attr("src"), outputPath).src;
+    $elem.attr("src", newSrc);
+    // If the image already has srcset defined,
+    // update that to use the image CDN as well.
+    // Note that this assumes the srcset paths use the same src as the image.
```

Comment:
can we not?
I feel like we should tell our authors _not_ to use `srcset`, you update it here, and we put the default web.dev-hosted image at the end (we talked about the fallback).

Reply:
chatted over video about this. expanding the comment to mention that some components like Hero and PostCard rely on this because they do srcsetting ahead of time. Eventually we'd like to remove that behavior from those components and have all the srcsetting done in this transform. That'll be phase 2.